backend/image/forms.py

```diff
@@ -31,14 +31,57 @@ def value_from_datadict(self, data, files, name):
 
 
 class ImageAdminForm(forms.ModelForm):
-    file = SVGAndImageFormField(widget=SVGAndImageInput)
+    file = SVGAndImageFormField(
+        widget=SVGAndImageInput, help_text="Upload an image file (supported formats: JPG, JPEG, PNG, GIF, SVG, WEBP)"
+    )
+
+    title = forms.CharField(
+        max_length=255, help_text="A descriptive title for the image that will be used for administrative purposes"
+    )
+
+    description = forms.CharField(
+        required=False,
+        widget=forms.Textarea,
+        help_text="A detailed description of the image content - useful for content management",
+    )
+
+    alt = forms.CharField(
+        max_length=255,
+        required=False,
+        help_text="Alternative text that describes the image - important for accessibility and SEO. "
+        "This text is displayed if the image fails to load and is read by screen readers.",
+    )
+
+    href = forms.URLField(
+        required=False,
+        help_text="The URL where the image should link to when clicked. "
+        "Leave empty if the image should not be clickable.",
+    )
+
+    rel = forms.CharField(
+        max_length=255,
+        required=False,
+        help_text='The relationship attribute for the link (e.g., "nofollow" for SEO, '
+        '"noopener noreferrer" for security when linking to external sites)',
```

Comment:
Even though `rel` is ir*rel*evant if the image is not clickable: perhaps it's sensible to define `noreferrer` as a default? Not all users will know what to do with this. BTW `noopener` is redundant if `noreferrer` is set, if [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer) may be believed.